src/test/java/cloudBankPages/ClientListPage.java

```diff
@@ -0,0 +1,35 @@
+package cloudBankPages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import static org.testng.Assert.assertTrue;
+
+public class ClientListPage {
+
+    WebDriver driver;
+    @FindBy(xpath = "//h1[text()='Client List']")
+    public WebElement clientListTitle;
+
+    public WebElement getAccountNumLocator(String text){
+        String accountNameLocator = "//table[@id='dataTable']//tbody/tr/td[text()='%s']";
+        return driver.findElement(By.xpath(String.format(accountNameLocator, text)));
+    }
+
+    public WebElement getClientNameLocator(String text){
+        String clientNameLocator = "//tbody/tr/td[text()='%s']/preceding-sibling::td[2]";
+        return driver.findElement(By.xpath(String.format(clientNameLocator, text)));
+    }
+
+    public ClientListPage(WebDriver driver){

```

Comment:
The constructor is usually placed as the very first "method"

Reply:
done